connect.yaml

```diff
@@ -3,27 +3,28 @@ deployAs:
     applicationType: service
     endpoint: /
     configuration:
-      securedConfiguration:
+      standardConfiguration:
         - key: CTP_PROJECT_KEY
           description: Commercetools project key
         - key: CTP_CLIENT_ID
           description: Commercetools client ID
-        - key: CTP_CLIENT_SECRET
-          description: Commercetools client secret
         - key: CTP_SCOPE
           description: Commercetools client scope
-        - key: CTP_REGION
+         - key: CTP_REGION
           description: Region of Commercetools project
         - key: CTP_AUTH_URL
           description: Commercetools Auth URL
         - key: CTP_API_URL
           description: Commercetools API URL
         - key: CTP_SESSION_URL
           description: Session API URL
+      securedConfiguration:
+        - key: CTP_CLIENT_SECRET
+          description: Commercetools client secret
   - name: enabler
     applicationType: service
     endpoint: /
     configuration:
-      securedConfiguration:
+      standardConfiguration:
         - key: CTP_REGION
           description: Region of Commercetools project
```

Comment:
```suggestion
```

Maybe we can remove this?

Reply:
Had a talk with connect team. Unfortunately now it is not supported to have zero environment variables at this moment. Therefore I would rather keep it, so it can be deployed by `connect`.

They are planning to make new release to support it. Once it is released, we would remove this variable from `enabler` configuration